custom/abt/reports/expressions.py

```diff
@@ -113,9 +113,9 @@ def _get_language(cls, item):
 
     @classmethod
     def _get_warning(cls, spec, item):
-        default = spec.get("warning", "")
+        default = unicode(spec.get("warning", u""))
         if cls._get_language(item) == "fra":
-            return spec.get("french", default)
+            return unicode(spec.get("french", default))
```

Comment:
Interesting. So you're saying our YAML parser returns byte strings when the string is ASCII-only, and Unicode strings when there are non-ASCII characters in the string. If that's true, then what you've done here should be fine, although that seems like a shaky thing to rely on and/or the YAML parser is doing something silly.

There *should* be a mode (note, I'm not saying there *is*) for the YAML parser that would make it return all Unicode strings, but that might not be easy to achieve.